apps/web/src/lib/inngest/process-bookmark.job.ts

```diff
@@ -29,15 +29,28 @@ export const processBookmarkJob = inngest.createFunction(
       const error = event.data.error;
 
       try {
-        await prisma.bookmark.update({
-          where: { id: bookmarkId },
-          data: {
-            status: "ERROR",
-            metadata: {
-              error: error.message,
+        await Promise.all([
+          prisma.bookmark.update({
+            where: { id: bookmarkId },
+            data: {
+              status: "ERROR",
+              metadata: {
+                error: error.message,
+              },
             },
-          },
-        });
+          }),
+          prisma.bookmarkProcessingRun.updateMany({
+            where: { 
+              bookmarkId: bookmarkId,
+              status: "STARTED",
+            },
+            data: {
+              status: "FAILED",
+              completedAt: new Date(),
+              failureReason: error.message,
+            },
+          }),
+        ]);
```

Comment:
update just the current run, the inngestRunId should be used to know which run is linkedin !